spring-cloud-starter-stream-source-sftp/pom.xml

```diff
@@ -17,6 +17,17 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-sftp</artifactId>
+			<version>5.0.7.BUILD-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-file</artifactId>
+			<version>5.0.7.BUILD-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+			<version>5.0.7.BUILD-SNAPSHOT</version>
```

Comment:
We can remove now all these explicit versions: the parent brings the proper one.